tests/test_client.ts

```diff
@@ -894,6 +1215,27 @@ it(
   }),
 );
 
+it(
+  'ai.dagexecute_ro negative testing',
```

Comment:
please add a fail test for `dataexecute_ro` with `scriptexecute`
this should not be possible

Reply:
done